src/__tests__/transmission_test.js

```diff
@@ -2,16 +2,25 @@
 import "babel-polyfill";
 
 import { Transmission, ValidatedEvent } from "../transmission";
+
+import http from "http";
 import net from "net";
+import superagent from "superagent";
+import superagentMocker from "superagent-mocker";
 
-let superagent = require("superagent");
-let mock = require("superagent-mocker")(superagent);
+let mock;
 
 describe("base transmission", () => {
+  beforeEach(() => (mock = superagentMocker(superagent)));
+  afterEach(() => {
+    mock.clearRoutes();
+    mock.unmock(superagent);
+  });
+
   it("will hit a proxy", done => {
     let server = net.createServer(socket => {
       socket.end();
-      server.unref();
+      server.close();
```

Comment:
should have the same effect, but I'd rather have things be clearer - we're not just dropping our ref to the server, we're _done_.  same below.